Add kafka inbound endpoint properties (#1114)
wso2/docs-mi: en/docs/reference/connectors/kafka-connector/kafka-inbound-endpoint-config.md

            <td>Offsets are committed automatically with a frequency controlled by the config.</td>
            <td><code>5000</code></td>
        </tr>
        <tr>
            <td><code>failure.retry.count</code></td>

>             <td>The offset set to the same record until the failure retry count exceeded.</td>

Reviewer: ```suggestion
            <td>The offset set to the same record until the failure retry count exceeds.</td>
```

            <td><code>False</code></td>
        </tr>
</table>

!!! Note

>     The `<property name="SET_ROLLBACK_ONLY" value="true"/>` should set in the fault sequence to poll the same record if in a failure case. The internal logic does not commit if the above property set and the offset set to the current record. The `failure.retry.count` used to control the same record polling in a failure scenario. The offset set to next discarding the current record after exceeding the retry count. The default value is `-1` means infinitely polling the same record in failure cases. The `<property name="SET_ROLLBACK_ONLY" value="true"/>` and `failure.retry.count` parameter only effective when `enable.auto.commit` set to `false`.

Reviewer: ```suggestion
    The `<property name="SET_ROLLBACK_ONLY" value="true"/>` property should be set in the fault sequence to poll the same record if in a failure case. The internal logic does not commit if the above property is set and the offset is set to the current record. The `failure.retry.count` parameter controls the polling of the same record during failure scenarios. If the retry count is exceeded, the offset is set to the next record, discarding the current one. The default value is `-1`, meaning the same record will be polled infinitely in failure cases. Both the `<property name="SET_ROLLBACK_ONLY" value="true"/>` and `failure.retry.count` parameters are effective only when `enable.auto.commit` is set to `false`.
```
